src/renderers/dom/client/eventPlugins/TapEventPlugin.js

```diff
@@ -20,19 +21,59 @@ var ViewportMetrics = require('ViewportMetrics');
 var isStartish = EventPluginUtils.isStartish;
 var isEndish = EventPluginUtils.isEndish;
 
+import type {TopLevelTypes} from 'EventConstants';
```

Comment:
I've been adding a space between `{}` but looks like the convention on www is not to (15k vs 500). Good thing you're helping on this effort so that I don't start diverging by mistake :p